scripts/discord_joey.py

```diff
@@ -0,0 +1,170 @@
+# coding: utf-8
```

Comment:
Maybe add a comment here that explains how to launch it and what kind of pointers need to be given (e.g. the model).